packages/tcc/build.sh

```diff
@@ -30,81 +33,22 @@ termux_step_post_get_source() {
 	fi
 }
 
-termux_step_pre_configure() {
-	cd $TERMUX_PKG_BUILDDIR
-	rm -rf _bin
-	mkdir _bin
-	cd _bin
-	local _ar="$TERMUX_HOST_PLATFORM-ar"
-	cat > "${_ar}" <<-EOF
-		#!$(command -v sh)
-		exec $(command -v $AR) "\$@"
-	EOF
-	chmod 0700 "${_ar}"
-	export PATH="$(pwd):$PATH"
-}
-
 termux_step_configure() {
-	unset CFLAGS CXXFLAGS
-
-	if [ "${TERMUX_ARCH}" = "arm" ] || [ "${TERMUX_ARCH}" = "i686" ]; then
-		ELF_INTERPRETER_PATH="/system/bin/linker"
-		ANDROID_LIB_PATH="/system/lib:/system/vendor/lib"
-	else
-		ELF_INTERPRETER_PATH="/system/bin/linker64"
-		ANDROID_LIB_PATH="/system/lib64:/system/vendor/lib64"
-	fi
+	# override default step
+        echo "running on : $(uname -a)"
```

Comment:
Was this meant to be left in here?
Or is this just a leftover debugging aid.

Reply:
Well, long story: Because of package.yml: "Setup arm and aarch64 CPU emulators" I got curious whether the "stage 1" tcc would maybe run in qemu.  As it turns out it doesn't,  it's always a [x86_64](https://github.com/termux/termux-packages/actions/runs/8041234944/job/21967219243#step:8:468) executable, regardless of the final package architecture.  Indeed we don't really have to know that, as long as there aren't any problems ...

Reply:
The qemu was never used anywhere. It also need a Android linker to work. So you can remove it.

Reply:
Unresolving, sinceits still there.